packages/db-base/lib/inMemFileDB.js

```diff
@@ -71,77 +71,84 @@ class InMemoryFileDB {
         }
         this.dataDir = this.dataDir.replace(/\\/g, '/');
 
-        // Create data directory
-        if (!fs.existsSync(this.dataDir)) {
-            fs.mkdirSync(this.dataDir);
+        this.datasetName = path.join(this.dataDir, this.settings.fileDB.fileName);
+        const parts = path.dirname(this.datasetName);
+        if (!fs.existsSync(parts)) {
+            fs.ensureDirSync(parts);
         }
```

Comment:
you can get rid of the existsSync - ensurDir does exactly that.

Reply:
Ok